monasca/templates/thresh-deployment.yaml

```diff
@@ -106,7 +106,7 @@ spec:
                 {{- end }}
                   key: "{{ .Values.thresh.mysql.password.secret_key | default "password" }}"
             - name: NO_STORM_CLUSTER
-              value: "true"
+              value: "{{ .Values.thresh.use_local }}"
```

Comment:
Does use_local need a default here?

Reply:
it has an implicit default in `monasca/values.yaml`